apps/vr-tests-react-components/package.json

```diff
@@ -27,6 +27,7 @@
     "@fluentui/react-card": "^9.0.14",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28109&commitId=d853a95077538b63e4c78578073299c05b476c11&env=prod)

✅ There was 1 screenshots added, 0 screenshots removed, 1944 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning (no decorator).scroll jumps (with context usage).chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>